docs/harness-attributes.md

```diff
@@ -0,0 +1,68 @@
+# Using harness attributes
+
+The Go harness has several attributes that you can use in your own project to control how it behaves. This document outlines some of the most common attributes that you will need to use when developing an application with it.
+
+> _NOTE: This document is a work-in-progress, and only a subset of all attributes have been documented. Please consider contributing._
+
+## Required attributes
+
+Some attribute values are required in order to create a working application. You will be asked to provide these whenever you create a project (`ws create-project ...`):
+
+* `module_name` - this will be the name of the Go module for your created application, e.g. `github.com/inviqa/repo-name`, it is stored in the generated `go.mod` file.
```

Comment:
```suggestion
* `module_name` - this will be the name of the Go module for your created application, e.g. `github.com/inviqa/repo-name`. This is stored in the generated `go.mod` file.
```